x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/rules_tables.tsx

```diff
@@ -5,6 +5,8 @@
  * 2.0.
  */
 
+/* eslint-disable complexity */
```

Comment:
Cyclomatic complexity of the `RulesTables` component exceeded the limit. I disabled it for now. I think a decent refactoring is needed for the whole table (maybe the whole page).

Reply:
yikes, we might want to file that in a tech debt ticket

Reply:
@dplumlee Agree, I'll do that. I'll add a few tickets and update my plan for 7.13+. I'm planning to focus on optimizing our rules-related endpoints now (https://github.com/elastic/kibana/issues/91265 is in fact about that). Once I'm done with the endpoints, I hope to switch to the client-side table implementation and this tech debt.

Thanks for reminding about being explicit and transparent 👍

Reply:
Here we go: https://github.com/elastic/kibana/issues/92160 and https://github.com/elastic/kibana/issues/92169